### PR TITLE
Add mosquitto

### PR DESCRIPTION
**Pull request by everm1nd**

### CODE REVIEW

**Comment by everm1nd** [comment]
@GContaldi @nicor88

File: docker-compose.yml [hunk 1]
```diff
@@ -4,4 +4,9 @@ mongodb:
   ports:
     - 27017:27017
   volumes:
-    - './data:/data/db'
\ No newline at end of file
+    - './data:/data/db'
```
Comment (GContaldi) [inline docker-compose.yml]: which db is using?

**Comment by GContaldi** [comment]
+1

**Review by GContaldi — APPROVED** [review]
+1

**Comment by nicor88** [comment]
+1

**Pull request merged**